controllers/board-game-review-controllers.js

```diff
@@ -83,3 +84,14 @@ exports.getUsers = (req, res, next) => {
         next(err);
     });
 };
+
+exports.deleteComment = (req, res, next) => {
+    const id = req.params.comment_id
+    removeComment(id)
+    .then((comment) =>{
+        res.status(204).send({comment})
```

Comment:
express will not send back a response body so one should not be included in the code. res.sendStatus(204) is fine here. 👍